### PR TITLE
[pulsar-proxy] Add CLI support to get pulsar-proxy stats

### PR DESCRIPTION
**Pull request by rdhabalia**
### Motivation
Note: This PR is top on #6473 
In PR: #6473 we have added REST-API to get stats for the pulsar-proxy.
However, we also need cli tool to call such api and retrieve stats.

### Modification
Added CLI for proxy-stats.

**Note:**
I will rebase this PR once, #6473 changes are reviewed and merged.

### CODE REVIEW

File: pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdProxyStats.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+
+@Parameters(commandDescription = "Operations to collect broker statistics")
```
Comment (codelipenghui) [inline pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdProxyStats.java]:
```suggestion
@Parameters(commandDescription = "Operations to collect proxy statistics")
```

File: pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdProxyStats.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    }
+
+    public void printStats(JsonElement json, boolean indent) throws IOException {
+        GsonBuilder builder = new GsonBuilder();
+        Gson gson = indent ? builder.setPrettyPrinting().create() : builder.create();
+        System.out.println(gson.toJson(json));
+    }
+
+    public CmdProxyStats(PulsarAdmin admin) {
+        super("broker-stats", admin);
```
Comment (codelipenghui) [inline pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdProxyStats.java]:
```suggestion
        super("proxy-stats", admin);
```

**Comment by jiazhai** [comment]
@rdhabalia Would you please handle @codelipenghui 's comments and rebased with latest master?

File: pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdProxyStats.java (hunk 2 above)
Comment (rdhabalia) [reply pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdProxyStats.java:68]: oops. good catch.let me fix it.

**Comment by rdhabalia** [comment]
@jiazhai @ @codelipenghui addressed the comments and also rebased. 👍

**Comment by sijie** [comment]
@codelipenghui can you review this pull request again?

**Comment by Anonymitaet** [comment]
@rdhabalia thanks for your great work. 

One quick question: have you added docs for both pulsar-admin and REST API?

**Pull request merged**